docpages/building/linux.md

```diff
@@ -37,12 +37,27 @@ Once installed to the `/usr/local` directory, you can make use of the library in
 g++ -std=c++17 mydppbot.cpp -o dppbot -ldpp
 ```
 
+If you are on Arch Linux you can add `/usr/local` to your enviorment variable:
+
+```bash
+export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+export CPLUS_INCLUDE_PATH=/usr/local/include:$CPLUS_INCLUDE_PATH
+```
+
+OR if you dont want to change your enviorment you can use add these flags while compiling.
```

Comment:
Or doesn't need to be full-caps here, just put `However, ...`.

Environment is spelt wrong here (you put `enviorment`). Replace `dont` with `don't` (Punctuation matters!). The correct sentence here is `However, if you don't want to change your environment settings, you can use these flags to compile:`

The sentence should also be changed to `you can use these flags` rather than `you can use add these flags`. It should be `use` as you're showing all the flags, not just the select ones.

Reply:
`you can use add` use add was type hehe sorry